lib/dev/ui/spinner.rb

```diff
@@ -31,7 +31,7 @@ class Task
           def initialize(title, &block)
             @title = title
             @thread = Thread.new do
-              cap = Dev::UI::StdoutRouter::Capture.new(with_frame_inset: false, &block)
+              cap = Dev::UI::StdoutRouter::Capture.new(self, with_frame_inset: false, &block)
```

Comment:
^ we have a test in the same file that doesn't use the parameter. also AFIK, you can pass as many/as few args to a block and it's never an error right away. https://gist.github.com/alanwushopify/95ff2a36cf07d7da962d8512e7e026d5